configuration/types.go

```diff
@@ -278,11 +278,11 @@ type DataConfiguration struct {
 	// been drained (if reconciliation is enabled).
 	ReconciliationDrainDisabled bool `json:"reconciliation_drain_disabled"`
 
-	// InactiveDiscrepencySearchDisabled is a boolean indicating if a search
-	// should be performed to find any inactive reconciliation discrepencies.
+	// InactiveDiscrepancySearchDisabled is a boolean indicating if a search
+	// should be performed to find any inactive reconciliation discrepancies.
 	// Note, a search will never be performed if historical balance lookup
 	// is disabled.
-	InactiveDiscrepencySearchDisabled bool `json:"inactive_discrepency_search_disabled"`
+	InactiveDiscrepancySearchDisabled bool `json:"inactive_discrepancy_search_disabled"`
```

Comment:
You need to change all the references where this is being used.

Reply:
This was updated in the b6b506e commit. However, for some reason, GitHub thinks it's an unverified merge.